### ISSUE TITLE
README and tests warnings

### ISSUE THREAD

**Issue by rrenka** [underworldcode/stripy#23]
I upgraded to Python 3.7, reinstalled stripy, and ran the tests again.
They work fine but issue DeprecationWarnings on time.clock.

The README.md file has references to python and pip commands.
Should they be changed to python3 and pip3 since Python 3.5 
is now specified as a requirement?

**Comment by lmoresi** [comment]
@brmather - while we are busy tidying the other issues, shall we fix the deprecated dependencies ? 

Also @brmather, have we introduced features that break under 2.7 in the most recent code as I have been using the notebooks with either kernel.

**Comment by lmoresi** [comment]
@rrenka - depending on the answer to the previous comment ... 

pip2 and pip3 ensure that the relevant python is used and do not work properly if the relevant python is not installed. pip by itself is supposed to work with whichever python is currently active. So we would usually leave this as pip because we **should** be able to work under 2.7. 

And this is a transition year for python in which we are all supposed to leave v2 behind and all this discussion will itself become deprecated.

**Comment by brmather** [comment]
I do believe stripy should be compatible with 2.7 and 3.x

A more general approach to installation with pip is:

```sh
python2 -m pip install stripy
python3 -m pip install stripy
```

This has been updated with 1243033ea5ed758e51a4142bebe3062cae52c198. I suppose it doesn't hurt to have legacy 2.7 despite it being sent to pasture in 2020.

**Comment by brmather** [comment]
DepreciationWarning(s) _should_ be fixed in 7fd0132f62aa90588c005115f708f45bcd989b6e.

For the purpose of the test scripts, `time.clock` and `time.time` are interchangeable. Except the latter is not (yet) depreciated.

**Comment by lmoresi** [comment]
OK - we are probably in a position to close this. 

@rrenka - feel free to re-open if related issues continue or it's not exactly working as advertised.
